packages/web-components/public/appointment-planner.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
```

Comment:
Appointment planner is a shared Svelte component between 2 new components Book a Viewing and Book a Valuation. Should be rendered by both. 

Also, each should copy their own index.html to the public dir at build - I added a script to do this in #785

Reply:
The public folder should not be committed